### DISCUSSION TITLE
just a backlink: I have added to reddit r/code_rosettas a link to this repo

### DISCUSSION THREAD

**Discussion by martin12333** [codethesaurus/codethesaur.us#750] [Ideas]
https://www.reddit.com/r/code_rosettas/comments/1epgrx8/code_thesaurus_the_polyglot_developer_reference/

https://github.com/code-rosettas

https://hyperpolyglot.org/

**Comment by geekygirlsarah** [comment]
Hi @martin12333! Thanks for the link. I didn't know about the subreddit but looking through it has some interesting links. I'll have to read up more!

And back in 2015 when I had the idea for Code Thesaurus and was trying to see if anyone had done something like this, Rosetta Code and Hyperpolyglot were two sites I had come across. They are great but just not entirely what I was looking for (though Hyperpolyglot was about the closest). They definitely helped inspire some ideas that I followed through on with it!

Thank you for sharing, I appreciate it!
